### DISCUSSION TITLE
IFTTT: Vocês usam?

### DISCUSSION THREAD

**Discussion by matheusml** [frontendbr/forum#757] [Perguntas e Respostas]
Galera, 

já tentei usar o IFTTT algumas vezes mas acabo abandonando depois de um tempo. Sinto que a ferramenta é extremamente útil e existe um grande valor nela que eu simplesmente não estou conseguindo extrair.

Vocês usam? Porque? Quais recipes?

Abraços!

**Comment by kvnol** [comment]
Eu uso para coisas básicas como publicar fotos do instagram no twitter, publicar o que estou ouvindo no Twitter e essas coisas, há um bom tempo até. Nunca senti necessidade de usar como uma ferramenta mais robusta.

**Comment by munizart** [comment]
Eu uso pra algumas recipes básicas, tipo notificar se vai chover, logar minha posição e horário quando apertar o botão (pra saber que horário o busão ta passando e em quais pontos).
O mais avançado que fiz foi uma recipe pra notificar quando um job grande termina de rodar no terminal.
Já considerei fazer algo com Arduino com ele tambem.

E a lá IFTTT estou criando um bot pra me noticar no celular conforme eu for configurando (lembretes, eventos, integração com webhooks etc)

A dica é que da pra configurar algumas recipes como webhook usando o canal maker.

**Comment by ghost** [comment]
Testei esses dias conectando meu Twitter ao Facebook e achei meio mancada o serviço exigir que o usuário clique cada vez para que o boot faça o compartilhamento dos tweets.

No mínimo eles podiam automatizar o processo de publicação, ai sim seria uma mão na roda.

**Comment by tcelestino** [comment]
@aicoutos, mas ai acho que é uma questão de "limitação" no uso da API's desses serviços. Se não estou enganado, a API do Twitter é bem restrita para postagem automática em sequências. O fato de você ter que ativar, pode ser considerado um "limitador" para essa prática.

---

Uso o IFTTT para três coisas:

1. Quando leio um post em um blog e marco como "Read Later" no feedly, ele automaticamente salva no Pocket.
2. Quando adiciono um vídeo na playlist "Watch Later" no YouTube, automaticamente ele manda para o Pocket com a tag "youtube"
3. Quando dou um like no vídeo (no YouTube), ele adiciona em uma planilha no Drive, assim acho mais prático para achar algo antigo (isso foi antes do YT implementar a playlist "Liked videos").
4. Quando faço um checkin no Foursquare, adiciona em uma planilha no Driver.

Queria poder usa-lo mais, inclusive com ações mais legais, mas é preciso tempo para usa-lo profundo. Eu recomendo 😄
